bin/www

```diff
@@ -68,9 +72,18 @@ function startServer() {
         }
     });
     server.on('listening', function() {
-        logger.debug(
-            `App now running on ${describeAddressBinding(server.address())}`
-        );
+        const msg = `App now running on ${describeAddressBinding(
+            server.address()
+        )}`;
+        logger.debug(msg);
+        if (notifier) {
+            notifier.notify({
+                title: 'blf-alpha',
```

Comment:
Wonder if we should renamed references to `blf-alpha` over to `tnlcommunityfund` at some point?